gramps/gui/editors/editplace.py

```diff
@@ -187,9 +187,14 @@ def _setup_fields(self):
 
     def set_latlongitude(self, value):
         try:
-            coma = value.index(', ')
-            longitude = value[coma+2:].strip().replace(',','.')
-            latitude = value[:coma].strip().replace(',','.')
+            # Bug 12349
+            parts = value.split(', ')
+            if len(parts) == 2:
+                longitude = parts[0].strip().replace(',', '.')
+                latitude = parts[1].strip().replace(',', '.')
```

Comment:
This is reversed! Latitude is the first field and longitude the second one according to the label above the text input.

See the result for a village in Belgium:

![image](https://user-images.githubusercontent.com/1447163/147589734-a896c54b-2320-476e-a490-a74f8b9ed4b4.png)

Reply:
A patch has been posted. 

See 
https://gramps.discourse.group/t/5-1-4-bug-in-gps-location-dialog/1731

Reply:
Yes only saw the existing patch #1290 after submitting mine #1320 😄 

Sorry for the noise.

Reply:
It is noise in the right place if it helps the next person looking here.